pkg/addons/data/DaemonSet.apps/openshift-azure-logging/mdsd.yaml

```diff
@@ -9,71 +9,68 @@ spec:
   selector:
     matchLabels:
       app: mdsd-shard
-  updateStrategy:
-    type: RollingUpdate
```

Comment:
We might want to add so it would not clean UpdateStrategy.

Reply:
FWIW, if updateStrategy is missing in apps/v1, [it gets defaulted to RollingUpdate](https://github.com/kubernetes/kubernetes/blob/3dbf0abddecdeef5956466b50d407399f0e447f2/pkg/apis/apps/v1/defaults.go#L75).